micromasters/tests/test_settings.py

```diff
@@ -1,17 +1,17 @@
 """
 Validate that our settings functions work, and we can create yaml files
 """
-
-import imp
 import os
 import sys
 import tempfile
 
 from django.conf import settings
 from django.core import mail
+from django.core.exceptions import ImproperlyConfigured
 from django.test import TestCase
 import mock
 import semantic_version
+from six.moves import reload_module
```

Comment:
I do not see `six` in the requirements. And since we decided to be python 3 only, I guess we can import directly from the right module.